mysite/polls/search.py

```diff
@@ -0,0 +1,102 @@
+import requests
+import re
+from more_itertools import unique_everseen
+
+
+def get_possible_movie_ids(movie_name):
+    url = "http://www.imdb.com/find?ref_=nv_sr_fn&q="+movie_name+"&s=all"
+    r = requests.get(url)
+    movie_id = re.findall(r'tt[0-9]+', r.text, re.M)
+    if not movie_id:
+        print "No movie matches this description"
+        exit()
```

Comment:
this will basically destroy your server. you could simply return empty result set instead